safe/utilities/test/test_styling.py

```diff
@@ -26,9 +26,9 @@
 from safe.test.utilities import (
```

Comment:
Ahah, thanks for spotting this file. This file can easily removed from InaSAFE and the linked functions. I will do a separate PR to see

Reply:
Thanks :)